share/VERSION.txt

```diff
@@ -1 +1 @@
-1.9.0-SNAPSHOT
```

Comment:
It think, this one should have been updated automatically. Did something went wrong after the release or the update of this file is not part of the related scripts?

Reply:
According to the docs: https://cwiki.apache.org/confluence/display/AVRO/How+To+Release
In step 8 this is still done by hand. I think I've missed this step.

Reply:
I see, thanks. Maybe it worth a JIRA to make the release less error prone.

Reply:
Fully agree. I've created two tickets to avoid PR's like these in the future:
https://jira.apache.org/jira/browse/AVRO-2406
https://jira.apache.org/jira/browse/AVRO-2407